lib/Selenium/Remote/Driver.pm

```diff
@@ -1008,6 +1008,46 @@ sub move_to {
     return shift->mouse_move_to_location(@_);
 }
 
+=head2 move_action
+
+ Description:
+    Move the mouse similarly to the mouse_move_to_location. But uses the
+    new webdrive actions API which is supported by geckodriver.
+
+ Output:
+    STRING -
+
+ Usage:
+    # element - the element to move to. Must be specified.
+    # xoffset - X offset to move to, relative to the center of the element. If not specified, the mouse will move to the middle of the element.
+    # yoffset - Y offset to move to, relative to the center of the element. If not specified, the mouse will move to the middle of the element.
+
+    print $driver->mouse_move_to_location(element => e, xoffset => x, yoffset => y);
+
+=cut
+
+sub move_action {
+    my ( $self, %args ) = @_;
+    my $params = {
+        actions => [{
+            type => "pointer",
+            id => 'my pointer move',
```

Comment:
This seems like an odd id -- I am guessing it is just a value that doesn't matter?

Reply:
yep